content/docs/1.17/operator.md

```diff
@@ -263,12 +263,13 @@ spec:
           topic: jaeger-spans
           brokers: my-cluster-kafka-brokers.kafka:9092
       ingester:
-        deadlockInterval: 5 # <2>
+        deadlockInterval: 5s # <2>
   storage:
     type: elasticsearch
     options:
       es:
         server-urls: http://elasticsearch:9200
+        #index-prefix: my-prefix
```

Comment:
Why comment? When published on the main site it looks like its leftover by mistake